docs/02-app/01-building-your-application/01-routing/03-linking-and-navigating.mdx

```diff
@@ -117,48 +124,55 @@ export default function Page() {
-- The router updates the URL in the browser's address bar.
-- The router avoids unnecessary work by re-using segments that haven't changed (e.g. shared layouts) from the [client-side cache](#client-side-caching-of-rendered-server-components). This is also referred to as [partial rendering](/docs/app/building-your-application/routing#partial-rendering).
-- If the [conditions of soft navigation](#conditions-for-soft-navigation) are met, the router fetches the new segment from the cache rather than the server. If not, the router performs a [hard navigation](#hard-navigation) and fetches the Server Component payload from the server.
-- If created, [loading UI](/docs/app/building-your-application/routing/loading-ui-and-streaming) is shown from the server while the payload is being fetched.
-- The router uses the cached or fresh payload to render the new segments on the client.
+### 1. Prefetching
 
-### Client-side Caching of Rendered Server Components
+Prefetching is a way to preload a route in the background before the user visits it.
 
-> **Good to know**: This client-side cache is different from the server-side [Next.js HTTP cache](/docs/app/building-your-application/data-fetching#caching-data).
+There are two ways routes are prefetched in Next.js:
 
-The new router has an **in-memory client-side cache** that stores the **rendered result** of Server Components (payload). The cache is split by route segments which allows invalidation at any level and ensures consistency across concurrent renders.
+- **`<Link>` component**: Routes are automatically prefetched as they become visible in the user's viewport. Prefetching happens when the page first loads or when it comes into view through scrolling.
+- **`router.prefetch()`**: The `useRouter` hook can be used to prefetched routes programmatically.
 
-As users navigate around the app, the router will store the payload of previously fetched segments **and** [prefetched](#prefetching) segments in the cache.
+The`<Link>`'s prefetching behavior is different for static and dynamic routes:
 
-This means, for certain cases, the router can re-use the cache instead of making a new request to the server. This improves performance by avoiding re-fetching data and re-rendering components unnecessarily.
+- [**Static Routes**](/docs/app/building-your-application/rendering/static-and-dynamic-rendering#static-rendering-default): `prefetch` defaults to `true`. The whole route is prefetched and cached.
+- [**Dynamic Routes**](/docs/app/building-your-application/rendering/static-and-dynamic-rendering#dynamic-rendering): `prefetch` default to `false`. Only the shared layout down until the first `loading.js` file is prefetched and cached for 30s. This reduces the cost of fetching an entire dynamic route, and it means you can show an [instant loading state](/docs/app/building-your-application/routing/loading-ui-and-streaming#instant-loading-states).
```

Comment:
`prefetch` default to `false`

This is not the case. The default is "auto" which is what is being described afterwards. 

`prefetch: false` fully disables prefetching.

Reply:
Ah, my apologies. I was going by what Seb mentioned here: https://github.com/vercel/next.js/issues/42991#issuecomment-1566339183

> If you don't specify a prefetch option, we'll infer a default by the page type. Since static pages are inherently somewhat stale and also cheap to load, we default to prefetch={true} for static page to default to best perf. For dynamic pages, we default to prefetch={false} to default to best cost savings.